liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -714,4 +716,17 @@ protected Map<String,File> getLibertyDirectoryPropertyFiles() {
         return new HashMap<String,File> ();
     }
 
+    protected void setContainerEngine(AbstractContainerSupportUtil util) throws PluginExecutionException {
+        String LIBERTY_DEV_PODMAN = "liberty.dev.podman";
+        Properties mergedProperties = project.getProperties();
+        mergedProperties.putAll(System.getProperties()); //System/command line properties will take precedence
+        if (!mergedProperties.isEmpty()) {
+            Object isPodman = mergedProperties.get(LIBERTY_DEV_PODMAN);
+            isPodman = mergedProperties.get(LIBERTY_DEV_PODMAN);
+            if (isPodman != null) {
+                util.setIsDocker(!(Boolean.parseBoolean(isPodman.toString())));
```

Comment:
First, why are you getting the property twice? Second, why not use `containsKey` to check for it? Finally, should we check for the value not being an actual `true` or `false` before calling `setIsDocker` and give some warning?